ceph/ceph_admin/restart.py

```diff
@@ -29,10 +29,10 @@ def restart(self: Union[OrchProtocol, DaemonProtocol], config: Dict):
         restart_init_time = datetime.datetime.utcnow()
         rc = self.op("restart", config)
         args = config.get("args")
-        verify = args.pop("verify", True)
-        if verify:
-            rc = self.check_service_restart(
-                service_name=args.get("service_name"),
-                restart_init_time=restart_init_time,
-            )
+        if args:
+            if args.get("verify", True):
+                rc = self.check_service_restart(
+                    service_name=args.get("service_name"),
+                    restart_init_time=restart_init_time,
+                )
```

Comment:
This can be re-written as 
```
	if args and args.get("verify", True):
            rc = self.check_service_restart(
                service_name=args.get("service_name"),
                restart_init_time=restart_init_time,
            )
```

Reply:
Done.